pinot-core/src/main/java/com/linkedin/pinot/core/indexsegment/generator/SegmentGeneratorConfig.java

```diff
@@ -285,6 +287,14 @@ public void setTimeColumnName(String timeColumnName) {
     _segmentTimeColumnName = timeColumnName;
   }
 
+  public int getSequenceId() {
+    return _sequenceId;
+  }
+
+  public void setSequenceId(int sequenceId) {
```

Comment:
please add a javadoc here saying that this method should be used instead of setPostFix if it is the sequece number that is to be added. thanks. also add a javadoc in setPostFix to use setSequenceNumber() instead, if sequence number is needed at the end of segment name